### ISSUE TITLE
[Bug] RuntimeError: Unknown Layer Type: Interpolate

### ISSUE THREAD

**Issue by tenbayserver** [openvinotoolkit/openvino#6752]
##### System information (version)

- OpenVINO=> 2021.1
- Operating System / Platform => ubuntu20.04 amd
- Compiler => gcc
- Problem classification => model load

##### Detailed description

i want to load a IR model on IECore like this:
```
// ...
net = ie.read_network(model=scrfd_model_xml, weights=scrfd_model_bin)
net_infer = ie.load_network(self.session, 'GPU')
// ...
```
it will take a runtime crash:
```
RuntimeError: Unknown Layer Type: Interpolate
```
but it not take the same error by loading on 'CPU'
```
//...
net_infer = ie.load_network(self.session, 'CPU')
```
##### Issue submission checklist

 - [ ] I report the issue, it's not a question
   
 - [x] I checked the problem with documentation, FAQ, open issues, Stack Overflow, etc and have not found solution
   
 - [ ] There is reproducer code and related data files: images, videos, models, etc.

**Comment by brmarkus** [comment]
Can you provide more details about your model, please? Is it your own model (and would it be possible to share with us?), or have you downloaded it from Open-Model-Zoo?
Were you able to convert it to IR, could you share how you converted it?
Can you reproduce the behavior when loading the model with sample-apps, like "benchmark_app"? Can you share how you use the app and its command-line parameter (like which precision FP16, FP32, INT8)?

OpenVINO lists "Interpolate" as supported operations:
- https://docs.openvinotoolkit.org/latest/openvino_docs_ops_image_Interpolate_1.html
- https://docs.openvinotoolkit.org/latest/openvino_docs_ops_image_Interpolate_4.html

Can you - if you cannot share your model - at least share a snipet within the XML-file like the example-snippet from the above mentioned URLs?

**Comment by vladimir-paramuzov** [comment]
@tenbayserver Interpolate operation was not implemented in 2021.1 release on GPU. Please try newer OpenVINO release

**Comment by tenbayserver** [comment]
> Can you provide more details about your model, please? Is it your own model (and would it be possible to share with us?), or have you downloaded it from Open-Model-Zoo?
> Were you able to convert it to IR, could you share how you converted it?
> Can you reproduce the behavior when loading the model with sample-apps, like "benchmark_app"? Can you share how you use the app and its command-line parameter (like which precision FP16, FP32, INT8)?
> 
> OpenVINO lists "Interpolate" as supported operations:
> 
> * https://docs.openvinotoolkit.org/latest/openvino_docs_ops_image_Interpolate_1.html
> * https://docs.openvinotoolkit.org/latest/openvino_docs_ops_image_Interpolate_4.html
> 
> Can you - if you cannot share your model - at least share a snipet within the XML-file like the example-snippet from the above mentioned URLs?

i am convert from [scrfd-onnx](https://github.com/deepinsight/insightface/tree/master/detection/scrfd) to IR [models.zip](https://github.com/openvinotoolkit/openvino/files/6861123/models.zip) on the openvino workbench web

**Comment by tenbayserver** [comment]
> @tenbayserver Interpolate operation was not implemented in 2021.1 release on GPU. Please try newer OpenVINO release

what the version of openvino can begin to support the 'Interpolate' operation? i want to evaluate it

**Comment by vladimir-paramuzov** [comment]
@tenbayserver I'd recommend 2021.3 or 2021.4.
